installation/linux.md

```diff
@@ -45,8 +45,12 @@ The manual installation through a platform independent archive file is suited fo
 Installation through a package repository is the recommended choice on Debian/Ubuntu derivatives.
 Alternatively resort to the [manual installation approach](#manual-installation).
 
-As long as openHAB 2 is in it's development state you may choose between the latest *Beta* release or a *Snapshot* build. The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), always including the latest changes to the openHAB 2 core and add-ons.
+First, add the openHAB 2 Bintray repository key to your package manager:
 
+```shell
+wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' | sudo apt-key add -
+```
+As long as openHAB 2 is in it's development state you may choose between the latest *Beta* release or a *Snapshot* build. The snapshot build is created [almost daily](https://bintray.com/openhab/apt-repo2/openhab2#release), always including the latest changes to the openHAB 2 core and add-ons.
```

Comment:
I was looking into a Markdown linter as Validator here on github. I should spend a few more minutes on that :)